app/controllers/users_controller.rb

```diff
@@ -19,9 +28,62 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by params[:id]
```

Comment:
because you loaded user by using load_user before_action
so dont need this line, remove it please

Reply:
done